Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_First_SSO_Access.yml

```diff
@@ -0,0 +1,1906 @@
+              complex:
+                root: Verdict
+            iscontext: true
+          right:
+            value:
+              simple: Malicious
+          ignorecase: true
+        - operator: greaterThanOrEqual
+          left:
+            value:
+              complex:
+                root: DBotScore
+                accessor: Score
+            iscontext: true
+          right:
+            value:
+              simple: "3"
+    - label: Suspicious
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: Verdict
+            iscontext: true
+          right:
+            value:
+              simple: Suspicious
+          ignorecase: true
+    - label: Suspicious - Ask the Manager
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: Verdict
+            iscontext: true
+          right:
+            value:
+              simple: Suspicious - Ask the Manager
+          ignorecase: true
+    - label: Non-malicious
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: Verdict
+            iscontext: true
+          right:
+            value:
+              simple: Non-malicious
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 267.5,
+          "y": 870
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "72":
+    id: "72"
+    taskid: a37362fe-c6c2-4357-849b-5636850ddd26
+    type: title
+    task:
+      id: a37362fe-c6c2-4357-849b-5636850ddd26
+      version: -1
+      name: Remediation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "135"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 677.5,
+          "y": 1920
+        }
+      }
+    note: false
+    timertriggers:
+    - fieldname: remediationsla
+      action: start
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "76":
+    id: "76"
+    taskid: e2409291-5dae-4067-8154-febfbf9ea338
+    type: title
+    task:
+      id: e2409291-5dae-4067-8154-febfbf9ea338
+      version: -1
+      name: Done
+      description: Closes the investigation.
+      type: title
+      iscommand: false
+      brand: Builtin
+    reputationcalc: 1
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -230,
+          "y": 3210
+        }
+      }
+    note: false
+    timertriggers:
+    - fieldname: remediationsla
+      action: stop
```

Comment:
Who said the remediation even started?

Reply:
Done. Add Section Section Header for  "Remediation Complete" with Remediation Complete stop timer